frontend/libs/services/src/lib/dk-mqtt-client.service.ts

```diff
@@ -1,27 +1,90 @@
-import { Injectable } from '@angular/core';
-import { IMqttMessage, IPublishOptions, MqttService } from 'ngx-mqtt';
-import { Observable, Subject } from 'rxjs';
+import {
+  DestroyRef,
+  effect,
+  inject,
+  Injectable,
+  untracked,
+} from '@angular/core';
+import { MqttService } from 'ngx-mqtt';
+import {
+  GameSchema,
+  PlayersSchema,
+  Table,
+  TablesResponseSchema,
+  TeamColor,
+} from '@dig-kick/models';
+import { GameStore, PlayerStore, TableStore } from '@dig-kick/store';
+import { takeUntilDestroyed, toSignal } from '@angular/core/rxjs-interop';
 
 @Injectable({
   providedIn: 'root',
 })
 export class DkMqttClientService {
-  private topics: { [topic: string]: Subject<string> } = {};
-
-  constructor(private _mqttService: MqttService) {}
-
-  subscribe(topic: string): Observable<string> {
-    if (!this.topics[topic]) {
-      this.topics[topic] = new Subject<string>();
-      this._mqttService.observe(topic).subscribe((message: IMqttMessage) => {
-        this.topics[topic].next(message.payload.toString());
-      });
-    }
-    this._mqttService.observe(topic);
-    return this.topics[topic].asObservable();
+  readonly tableStore = inject(TableStore);
+  readonly gameStore = inject(GameStore);
+  readonly playerStore = inject(PlayerStore);
+
+  tablesRaw = toSignal(this._mqttService.observe('tables'));
+
+  scoreRaw = toSignal(this._mqttService.observe(`api/player/all`));
```

Comment:
You just load all players? Shouldn't it only be the players relevant to the current game?

Reply:
This loads all players for the scoreboard